src/stories/design-system/IntroductionToCreatingStories.stories.mdx

```diff
@@ -0,0 +1,222 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Introduction To Creating Stories" />
+
+# Creating stories of your own
+
+## We have a standardized way now of creating stories! It is encouraged that we follow proper documentation and be as informative as we can when creating our stories. 
+
+1. #### Create a component named ```YourCompentName.stories.tsx``` under you component folder.
+
+2. #### Add the necessary imports for your story
+```jsx
+// Some necessary imports
+import React from "react";
+import {ComponentStory, ComponentMeta} from "@storybook/react";
+import {withDesign} from "storybook-addon-designs";
+// Your component
+import BackgroundIcon from "./BackgroundIcon";
+// Other imports. In this case, we need to import the icons
+import {faSearch, faStar} from "@fortawesome/pro-solid-svg-icons";
+import {faStar as faStarLight} from "@fortawesome/pro-light-svg-icons";
+```
+
+3. #### Add a component meta. This will include a ```title, component, and decorators```. Here is an example:
+More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+```jsx
+export default {
+  title: "Components/BackgroundIcon",
+  component: BackgroundIcon,
+  decorators: [withDesign],
+} as ComponentMeta<typeof BackgroundIcon>;
+```
+
+4. #### In your story file, before the component meta, create a summary of the component as a short description, some context as to why the component was created, as well as its figma link
+```jsx
+const summary = "##Summary\nThe Background icon component may contain various icons to visualize or to give information as an image.\n\n";
```

Comment:
Suggestion; instead of putting markdown in JS strings, should we not load it from a markdown file to allow for easier editing?